app/script_variant.py

```diff
@@ -0,0 +1,80 @@
+import jautils
+
+from unidecode import unidecode
+from google.appengine.api import memcache
+
+import re
+import logging
+
+def read_dictionary(file_name):
+    dictionary = {}
+    for line in open(file_name, 'r'):
+        kanji, hiragana = line[:-1].split('\t')
```

Comment:
Could you comment what's the last character in the line that we need to strip off?